core/Tracker/RequestSet.php

```diff
@@ -145,41 +133,6 @@ protected function getAllSiteIdsWithinRequest()
         return array_values(array_unique($siteIds));
     }
 
-    // TODO maybe move to response? or somewhere else? not sure where!
-    public function shouldPerformRedirectToUrl()
-    {
-        if (!$this->hasRedirectUrl()) {
-            return false;
-        }
-
-        if (!$this->hasRequests()) {
-            return false;
-        }
-
-        $redirectUrl = $this->getRedirectUrl();
-        $host        = Url::getHostFromUrl($redirectUrl);
```

Comment:
`use` statement for `Url` class could be removed now...

Reply:
fixed